packages/shared/src/components/fields/Dropdown.tsx

```diff
@@ -114,9 +114,7 @@ export function Dropdown({
         aria-expanded={isVisible}
       >
         {icon}
-        <span className="flex flex-1 mr-1 truncate">
-          {options[selectedIndex]}
-        </span>
+        <span className="flex flex-1 truncate">{options[selectedIndex]}</span>
```

Comment:
I think we need the margin right for the arrow icon.
Perhaps it's better to remove the parent container set width, not sure why we needed that.